### PR TITLE
Added BIGINT type to mapper

### PR DESCRIPTION
**Pull request by jarno-steeman**

### CODE REVIEW

**Comment by jarno-steeman** [comment]
In favor of this (too hasty made) PR https://github.com/mickhansen/graphql-sequelize/pull/16

File: src/typeMapper.js [hunk 1]
```diff
@@ -31,7 +32,11 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
 
   if (sequelizeType instanceof BOOLEAN) return GraphQLBoolean;
   if (sequelizeType instanceof FLOAT) return GraphQLFloat;
-  if (sequelizeType instanceof INTEGER) return GraphQLInt;
+
+  if (sequelizeType instanceof INTEGER ||
+      sequelizeType instanceof BIGINT) {
```
Comment (mickhansen) [inline src/typeMapper.js]: what about the issue with `BIGINT` sometimes being too large to express as a javascript Number?

**Comment by jarno-steeman** [comment]
It will not break since GraphQLInt has a check for this, but will return null if it's too large for JS to handle (safely). I'd say the only sollution would be to make a custom GraphQL type for this, but it would have to work with strings.

**Comment by mickhansen** [comment]
Released as `v0.8.3`

**Pull request merged**